apis/src/main/java/org/apache/rocketmq/apis/exception/ClientException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.rocketmq.apis.exception;
+
+public class ClientException extends Exception {
```

Comment:
it seems this is the root Exception for all the rocketmq client exception. when we will throw this exception.
if maintainer want to add an exception which kind of exception can extend this clientException.

add java doc will be more formal

Reply:
Yes, we add `abstract` qualifier into `ClientException`, which makes it could be constructed directly, and java docs also has been added.